client/src/Components/NavigationBar.js

```diff
@@ -0,0 +1,185 @@
+import Typography from "@material-ui/core/Typography";
+import DashboardRoundedIcon from "@material-ui/icons/DashboardRounded";
+import CalendarTodayOutlinedIcon from "@material-ui/icons/CalendarTodayOutlined";
+import AddIcon from "@material-ui/icons/Add";
+import IconButton from "@material-ui/core/IconButton";
+import { NavLink } from "react-router-dom";
+import Avatar from "@material-ui/core/Avatar";
+import Menu from "@material-ui/core/Menu";
+import MenuItem from "@material-ui/core/MenuItem";
+import { DropzoneDialog } from "material-ui-dropzone";
+
+const useStyles = makeStyles((theme) => ({
+  appBar: {
+    background: "#FFFFFF",
+    boxShadow: "none",
+    flexDirection: "row",
+    alignItems: "center",
+    height: "85px",
+  },
+  toolBar: {
+    justifyContent: "space-between",
+    alignItems: "center",
+    flexGrow: 2,
+  },
+  buttonNotSelected: {
+    margin: theme.spacing(1),
+    color: "#666666",
+  },
+  selectedButton: {
+    margin: theme.spacing(1),
+    color: "#759DFD",
+  },
+  title: {
+    color: "black",
+  },
+  buttonText: {
+    fontSize: 14,
+    fontWeight: "bold",
+  },
+  buttonIcons: {
+    marginRight: "8px",
+  },
+  createButton: {
+    backgroundColor: "#759DFD",
+    marginRight: "40px",
+    width: "175px",
+    "&:hover": {
+      backgroundColor: "#759DFD",
+    },
+  },
+  createButtonText: {
+    color: "#FFFFFF",
+    fontWeight: "bold",
+    padding: "10px",
+  },
+  createButtonIcon: {
+    color: "#FFFFFF",
+  },
+  userIcon: {
+    width: "68px",
+    height: "68px",
+    flexShrink: 3,
+  },
+}));
+
+const NavigationBar = () => {
+  const initialStateDropFile = {
+    open: false,
+    files: [],
+  };
+
+  const [anchorEl, open] = React.useState(null);
+
+  const handleClickAvatarMenu = (event) => {
+    open(event.currentTarget);
+  };
+  const handleCloseAvatarMenu = () => {
+    open(null);
+  };
+
+  const [stateDropFile, setStateDropFile] = React.useState(
+    initialStateDropFile
+  );
+
+  const handleOpenDropFile = () => {
+    setStateDropFile({
+      ...stateDropFile,
+      open: true,
+    });
+  };
+
+  const handleCloseDropFile = () => {
+    setStateDropFile({
+      ...stateDropFile,
+      open: false,
+    });
+
+    handleCloseAvatarMenu();
+  };
+
+  const handleSaveDropFile = (files) => {
+    setStateDropFile({
+      ...stateDropFile,
+      files: files,
+      open: false,
+    });
+  };
+
+  const classes = useStyles();
+
+  return (
+    <AppBar position="static" className={classes.appBar}>
+      <Toolbar className={classes.toolBar}>
+        <Box>
+          <Button>
+            <Typography variant="title">
+              <img src="/images/logo.png" alt="bug" />
+            </Typography>
+          </Button>
+        </Box>
+        <Box>
+          <Button
+            className={classes.buttonNotSelected}
+            component={NavLink}
+            to="/"
+            activeClassName={classes.selectedButton}
```

Comment:
is activeClassName a button MUI prop? i was not able to find it on the MUI documentation. Not really sure how it works in this case. we may want to potentially discuss using a boolean key to conditionally handle the change in class names if this approach is not suitable

Reply:
NavLink utilizes activeClassName to change the appearance of itself. In this case Button adopts NavLink through the component property (component={NavLink}), so Button also adopts the activeClassName property so that Button can change its own appearance whenever it is active. It is necessary in order to make the button's text/icon blue when on that page.